lib/src/json_schema/validator.dart

```diff
@@ -61,6 +61,45 @@ class Instance {
 
   @override
   toString() => data.toString();
+
+  @override
+  bool operator ==(Object other) => other is Instance && this.path == other.path;
```

Comment:
Should we check whether `other != null` too?

Reply:
I don't think we need to because `null !is Instance`.